packages/core/admin/admin/src/components/AuthenticatedApp/index.js

```diff
@@ -78,15 +78,17 @@ const AuthenticatedApp = () => {
 
   const shouldShowLoader = isLoading || shouldShowNotDependentQueriesLoader;
 
+  const adminUserId = userInfo ? hashAdminUserEmail(userInfo) : '';
```

Comment:
I must have not changed it. In the early version I wasn't handling exceptions in the function, so had a ternary here. It's not needed anymore for sure, I'll change it

Reply:
I think I forgot to delete this, since first version of the `hashAdminUserEmail` didn't handle absence of `userInfo`. It does now, so ternary not necessary